.gitignore

```diff
@@ -31,6 +31,9 @@ oscn.tar
 # Ignore Data Files
 /lib/data/*
 
+# Ignore large cassetts for testing
+/spec/cassettes/large
```

Comment:
Added some DOC E2E tests (which can be recorded as cassettes)
Not actually committing the cassettes though since they're large, and skipping the tests by default.